llm/llama-2/chatbot-hf.yaml

```diff
@@ -0,0 +1,40 @@
+resources:
+  accelerators: A100:1
+  disk_size: 1024
+  disk_tier: high
+
+setup: |
+  conda activate chatbot
+  if [ $? -ne 0 ]; then
+    conda create -n chatbot python=3.9 -y
+    conda activate chatbot
+  fi
+
+  # Install dependencies
+  pip install git+https://github.com/lm-sys/FastChat.git
+  # Need the latest transformers to support 70B model
+  pip install git+https://github.com/huggingface/transformers.git
+
+  python -c "import huggingface_hub; huggingface_hub.login('${HF_TOKEN}')"
+
+run: |
+  conda activate chatbot
+  
+  echo 'Starting controller...'
+  python -u -m fastchat.serve.controller > ~/controller.log 2>&1 &
+  sleep 10
+  echo 'Starting model worker...'
+  python -u -m fastchat.serve.model_worker \
+            --model-path meta-llama/Llama-2-${MODEL_SIZE}b-chat-hf \
+            --num-gpus $SKYPILOT_NUM_GPUS_PER_NODE 2>&1 \
+            | tee model_worker.log &
+
+  echo 'Waiting for model worker to start...'
+  while ! `cat model_worker.log | grep -q 'Uvicorn running on'`; do sleep 1; done
+
+  echo 'Starting gradio server...'
+  python -u -m fastchat.serve.gradio_web_server --share | tee ~/gradio.log
+
+envs:
```

Comment:
Nit: `envs` before `setup` or `resources` may make it easier to spot.

Reply:
Sounds good. Done. Thanks!